src/index.js

```diff
@@ -1,8 +1,11 @@
-import pageViewTracker from "./analytics/pageViewTracker/pageViewTracker.ts";
-import formTracker from "./analytics/formTracker/formTracker.ts";
-import navigationViewTracker from "./analytics/navigationViewTracker/navigationViewTracker.ts";
+import { Analytics } from "./analytics/core/core.ts";
 
-pageViewTracker();
-formTracker();
-navigationViewTracker();
-console.log("running");
+window.DI = window.DI || {};
+(function (DI) {
+  "use strict";
+  function initAnalytics({ ga4ContainerId }) {
```

Comment:
Please can I double check, are we using modern javascript conventions for ES6?

e.g 
`const initAnalytics = ({ ga4ContainerId }) => {`

Reply:
Good question.
It's the starting point of our library and when webpack+babel create the es5 compiled file (npm build - creation of the analytics.js), I don't know if by using es6 in this file we will not break the build.
Will create a spike ticket about it.